jobs/jobs/run_job_funcs.py

```diff
@@ -19,27 +21,20 @@ async def run_extraction_job(
     """Runs ExtractionJob - creates init_args
     and sends it to the Pipeline Manager"""
 
-    if isinstance(
-        job_to_run.pipeline_id, str
-    ) and not job_to_run.pipeline_id.endswith(":airflow"):
-        raise HTTPException(
-            status_code=400,
-            detail="Wrong pipeline value.",
-        )
-
     db_service.update_job_mode(db, job_to_run, schemas.JobMode.Automatic)
     converted_files_data = utils.convert_files_data_for_inference(
         all_files_data=job_to_run.all_files_data,
         job_id=job_to_run.id,
         output_bucket=current_tenant,
     )
 
-    await utils.execute_pipeline(
+    logger.info("Starting external pipeline")
```

Comment:
Can we provide more info in this log or make it debug?

Reply:
What would you suggest here to add? Feel free to add commit

Reply:
`Starting external pipeline %s for job %s of %s tenant, %s engine`, job_to_run.pipeline_id, job_to_run.id, current_tenant, job_to_run.pipeline_engine

And Id make it debug tbh